onnxruntime/core/framework/debug_node_inputs_outputs_utils.cc

```diff
@@ -491,11 +561,20 @@ void DumpNodeInputs(
             const bool is_shape_set = (dump_options.dump_flags & NodeDumpOptions::DumpFlags::Shape) != 0;
             PrintIf(is_shape_set, MakeString(" Shape: ", shape, "\n"));
 
-            if ((dump_options.dump_flags & NodeDumpOptions::DumpFlags::InputData) != 0) {
+            if ((dump_options.dump_flags & NodeDumpOptions::DumpFlags::InputData) != 0 || check_half_overflow) {
```

Comment:
Seems that this blocklist is on the node level. Why not on the node type level? We only benchmark a few data, this skiplayer node type has potential to break with some data that is not in benchmark. Why not block this node type?

Reply:
Once you have node level block list, you can decide whether it is better to extend to node type block list.
Block list on node level has it pros and cons. The pros: it could achieve better performance when only a subset (like 50% SkipLayerNorm nodes). The cons: potential break with new data (the risk is smaller if you test more samples and use lower threshold).

Reply:
I also output list of operators in case user wants to use op_block_list.